src/ppca.jl

```diff
@@ -178,7 +171,9 @@ function MultivariateStats.fit{T<:AbstractFloat}(::Type{PPCA}, X::DenseMatrix{T}
     d, n = size(X)
 
     # process mean
-    mv = MultivariateStats.preprocess_mean(X, mean)
+    mv = preprocess_mean(X, mean)
+    (isempty(mv) || length(mv) == d) ||
+            throw(DimensionMismatch("Dimensions of weight matrix and mean are inconsistent."))
```

Comment:
Too much indentation here. I'd just turn this into an explicit `if` block; I think it'd read a little nicer.